src/xrEngine/CustomHUD.h

```diff
@@ -20,7 +20,7 @@ ENGINE_API extern Flags32 psHUD_Flags;
 class IGameObject;
 
 class ENGINE_API CCustomHUD : public FactoryObjectBase, public IEventReceiver,
-                              public CUIResetNotifier
+                              public CUIResetAndResolutionNotifier
```

Comment:
How is this relevant to MS specific types removal?

Reply:
Merge conflict